metriq-gym/v0.4/ibm/results.json

```diff
@@ -1,4 +1,33 @@
 [
+  {
+    "app_version": "0.4.2.dev7+g3343a86ca",
+    "timestamp": "2025-10-19T14:06:47.295695",
+    "suite_id": null,
+    "job_type": "BSEQ",
+    "results": {
+      "values": {
+        "largest_connected_size": 152.0,
+        "fraction_connected": 0.9743589743589743
+      },
+      "uncertainties": {
+        "largest_connected_size": 0.0,
+        "fraction_connected": 0.0
```

Comment:
hmmm I think we should have a special value for uncertainties that are not calculated. `0.0` makes it look like they were calculated as 0.0, which is misleading. I will add that to the framework tomorrow.

Reply:
@vprusso see https://github.com/unitaryfoundation/metriq-gym/pull/599